package.json

```
@@ -11,19 +11,19 @@
 	"exports": {
 		".": {
 			"module": "./dist/preact-router.module.js",
-			"import": "./dist/preact-router.module.js",
+			"import": "./dist/preact-router.mjs",
 			"require": "./dist/preact-router.js"
 		},
 		"./package.json": "./package.json",
 		"./match": {
 			"module": "./match/index.module.js",
-			"import": "./match/index.module.js",
+			"import": "./match/index.mjs",
 			"require": "./match/index.js"
 		},
 		"./match/package.json": "./match/package.json"
 	},
 	"scripts": {
-		"build": "microbundle -f es --no-generateTypes && microbundle src/cjs.js -f cjs,umd --no-generateTypes && (cd match && npm run build)",
+		"build": "microbundle -f es --no-generateTypes && copyfiles dist/preact-router.module.js dist/preact-router.mjs && microbundle src/cjs.js -f cjs,umd --no-generateTypes && (cd match && npm run build && copyfiles index.module.js index.mjs)",
```

```suggestion
		"build": "microbundle -f es --no-generateTypes && microbundle src/cjs.js -f cjs,umd --no-generateTypes && (cd match && npm run build)",
		"postbuild": "cp dist/preact-router.module.js dist/preact-router.mjs && cp match/index.module.js match/index.mjs",
```

[`copyfiles` pretty importantly copies into a **directory**](https://www.npmjs.com/package/copyfiles). What you have written will make a directory `dist/preact-router.mjs` and copy `preact-router.module.js` into it, i.e.:

```
dist/
--preact-router.mjs/
----preact-router.module.js
--preact-router.module.js
--...
```